Dockerfile.xdmod.init

```diff
@@ -0,0 +1,46 @@
+FROM moc-xdmod-base
+
+#COPY assets /tmp/assets
+#RUN /tmp/assets/copy-caches.sh
+#COPY bin /root/bin
+
+WORKDIR /root
+
+
+# This installs a barebones 9.5 instance of XDmod
```

Comment:
This comment doesn't match version of xdmod actually installed by the Dockerfile.

Reply:
done